pages/art-pieces/[slug].js

```diff
@@ -0,0 +1,20 @@
+import ArtPieceDetails from "@/components/ArtPieceDetails";
+import { useRouter } from "next/router";
+export default function ArtPieceDetailsPage({ data }) {
```

Comment:
Maybe you can find a more descriptive name than `data`.

Reply:
Later we will change it .. promise :)